@nordicjm 

I am encountering build failures in the "Run tests with twister / twister-build(2) (pull_request_target)

```
error log:
INFO    - The following issues were found (showing the top 10 items):
INFO    - 1) samples/psa/its/sample.psa.its.tfm on mps3/corstone300/fvp/ns error (Build failure)
INFO    - 2) samples/psa/persistent_key/sample.psa.persistent_key.tfm on mps3/corstone300/fvp/ns error (Build failure)
INFO    - 3) samples/psa/its/sample.psa.its.tfm on mps3/corstone310/fvp/ns error (Build failure)
INFO    - 4) samples/psa/persistent_key/sample.psa.persistent_key.tfm on mps3/corstone310/fvp/ns error (Build failure)
INFO    - 
INFO    - To rerun the tests, call twister using the following commandline:
INFO    - west twister -p <PLATFORM> -s <TEST ID>, for example:
INFO    - 
INFO    - west twister -p mps3/corstone310/fvp/ns -s samples/psa/persistent_key/sample.psa.persistent_key.tfm
INFO    - or with west:
INFO    - west build -p -b mps3/corstone310/fvp/ns samples/psa/persistent_key -T sample.psa.persistent_key.tfm
INFO    - -+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
INFO    - Run completed

Error: Error: failed to run script step: command terminated with non-zero exit code: Error executing in Docker Container: 1
Error: Process completed with exit code 1.
Error: Executing the custom container implementation failed. Please contact your self hosted runner administrator.
```

I am unable to determine how these build failures are related to the current patch. Could you please help clarify or provide guidance on troubleshooting these issues?

--- a/dts/arm/rpi_pico/override.dtsi
+++ b/dts/arm/rpi_pico/override.dtsi
@@ -1,0 +1,7 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/*
+ * File intentionally left blank. Will be used when there is no other
+ * higher-priority override.dtsi file in use.
+ */

--- a/dts/arm/rpi_pico/rp2040.dtsi
+++ b/dts/arm/rpi_pico/rp2040.dtsi
@@ -10,9 +10,17 @@
 #include <zephyr/dt-bindings/clock/rpi_pico_clock.h>
 #include <zephyr/dt-bindings/i2c/i2c.h>
 #include <zephyr/dt-bindings/regulator/rpi_pico.h>
+#include <zephyr/dt-bindings/reset/rpi_pico_reset.h>
 #include <mem.h>
 
-#include "rpi_pico_common.dtsi"
+#include <arm/rpi_pico/override.dtsi>
+/*
+ * This value can be overridden at the board level or in an application specific
+ * override.dtsi file.
+ */
+#ifndef RPI_PICO_DEFAULT_IRQ_PRIORITY
+#define RPI_PICO_DEFAULT_IRQ_PRIORITY 3
+#endif
 
 / {
 	aliases {

--- a/include/zephyr/dt-bindings/reset/rpi_pico_reset.h
+++ b/include/zephyr/dt-bindings/reset/rpi_pico_reset.h
@@ -4,10 +4,6 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#ifndef RPI_PICO_DEFAULT_IRQ_PRIORITY
-#define RPI_PICO_DEFAULT_IRQ_PRIORITY 3
-#endif
-
 #define RPI_PICO_RESETS_RESET_ADC		0
 #define RPI_PICO_RESETS_RESET_BUSCTRL		1
 #define RPI_PICO_RESETS_RESET_DMA		2

